### ISSUE TITLE
GENERATE_HTMLHELP with \msc & \endmsc is broken because of absolute paths in index.hhp since 1.9.3

### ISSUE THREAD

**Issue by XiaoP76** [doxygen/doxygen#10754]
**Describe the bug**
If I use \msc & \endmsc tags in a C++ comment and use GENERATE_HTMLHELP=YES, Doxygen fails with the error message:
```
Running html help compiler...
error: failed to run html help compiler on index.hhp
```
Running the hhc compiler manually reveals the problem:
```
d:\bug1.9.3>hhc html\index.hhp
Microsoft HTML Help Compiler 4.74.8702
Compiling d:\bug1.9.3\html\KCASystem.chm
HHC5003: Error: Compilation failed while compiling bug1.9.3\html\msc_inline_mscgraph_1.png.
The following files were not compiled:
bug1.9.3\html\msc_inline_mscgraph_1.png
Unable to open html\index.hhp.
```
The reason is that Doxygen 1.9.3 generates an absolute path in the index.hhp, e.g.
```
[FILES]
d:/bug1.9.3/./html\msc_inline_mscgraph_1.png
```
Doxgen 1.9.2 with the same Doxyfile generates a relative path:
```
[FILES]
msc_inline_mscgraph_1.png
```

**To Reproduce**
Extract the attached zip file [bug1.9.3.zip](https://github.com/doxygen/doxygen/files/14647723/bug1.9.3.zip)
and run
`doxygen Manual.doxygen`

**Expected behavior**
Expected is that the hhc compiler can create the chm file without an error even if \msc comments are used.

**Version**
Version 1.9.2 is the last one that works as expected.
Versions starting from 1.9.3 till the current 1.10.0 show the bug.

**Comment by albert-github** [comment]
Looks like that for the MSC graphs the path has been added as well and this is not supported by the hhc compiler.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #10755

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.11.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).

**Comment by XiaoP76** [comment]
I verified it and it's fixed in 1.11.0.
Thank you :-)
